langchain-core/src/runnables/base.ts

```diff
@@ -2068,6 +2068,10 @@ export class RunnableLambda<RunInput, RunOutput> extends Runnable<
       for await (const chunk of stream) {
         yield chunk;
       }
+    } else if (isAsyncIterator(output)) {
```

Comment:
Do we want to consume generators as well?

Reply:
yep good point, added